nucypher/cli/commands/ursula.py

```diff
@@ -342,7 +345,7 @@ def forget(general_config, config_options, config_file):
     emitter = setup_emitter(general_config, config_options.worker_address)
     _pre_launch_warnings(emitter, dev=config_options.dev, force=None)
     ursula_config = config_options.create_config(emitter, config_file)
-    forget(emitter, configuration=ursula_config)
+    forget_nodes(emitter, configuration=ursula_config)
```

Comment:
Worrying that it wasn't discovered before... a test for `ursula forget` would be nice (perhaps not in this PR)

Reply:
I am right there with you. Opened #2083